biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblReader.java

```diff
@@ -0,0 +1,147 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+
+import java.io.*;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * This class should process the data of embl file
+ * @since 5.0.0
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblReader {
+
+    private StringBuilder sequence = new StringBuilder("");
+
+    public EmblReader() {
```

Comment:
This no-arg public constructor can be removed

Reply:
done.